File: Peaker.TimeManagment/App/src/Models/UserListItem.ts
export class UserListItem {
    AccountingName: string;
    Email: string;
    UserDetailId: number;

    constructor(accountingName: string, email: string, userDetailId: number) {
        this.AccountingName = accountingName;
        this.Email = email;
        this.UserDetailId = userDetailId;
    }
}

File: Peaker.TimeManagment/Models/View/UserListItem.cs
using System.Collections.Generic;
using System.Data;

namespace Peaker.TimeManagment.Models.View
{
    public class UserListItem
    {
        public string AccountingName { get; set; }
        public string Email { get; set; }
        public int UserDetailId { get; set; }

        public static UserListItem UserListItemFactory(IDataRecord record)
        {
            return new UserListItem
            {                           
                AccountingName = record.GetString(record.GetOrdinal("AccountingName")),
                Email = record.GetString(record.GetOrdinal("Email")),
                UserDetailId = record.GetInt32(record.GetOrdinal("UserDetailId")),
            };
        }
    }
}